Repair CodeQL workflow

The CodeQL workflow is defined in .github/workflows/codeql-analysis.yml 

The workflow fails with error:
```
 Error: 1-04 10:18:38] [ERROR] Spawned process exited abnormally (code 255; tried to run: [/opt/hostedtoolcache/CodeQL/0.0.0-20211208/x64/codeql/javascript/tools/autobuild.sh])
  Error: 1-04 10:18:38] [ERROR] Spawned process exited abnormally (code 255; tried to run: [/opt/hostedtoolcache/CodeQL/0.0.0-20211208/x64/codeql/javascript/tools/autobuild.sh])
  A fatal error occurred: Exit status 255 from command: [/opt/hostedtoolcache/CodeQL/0.0.0-20211208/x64/codeql/javascript/tools/autobuild.sh]
  Error: The process '/opt/hostedtoolcache/CodeQL/0.0.0-20211208/x64/codeql/codeql' failed with exit code 2
  Error: The process '/opt/hostedtoolcache/CodeQL/0.0.0-20211208/x64/codeql/codeql' failed with exit code 2
      at toolrunnerErrorCatcher (/home/runner/work/_actions/github/codeql-action/v1/lib/toolrunner-error-catcher.js:94:19)
      at processTicksAndRejections (internal/process/task_queues.js:93:5)
      at async Object.extractScannedLanguage (/home/runner/work/_actions/github/codeql-action/v1/lib/codeql.js:491:13)
      at async createdDBForScannedLanguages (/home/runner/work/_actions/github/codeql-action/v1/lib/analyze.js:78:13)
      at async finalizeDatabaseCreation (/home/runner/work/_actions/github/codeql-action/v1/lib/analyze.js:95:5)
      at async runFinalize (/home/runner/work/_actions/github/codeql-action/v1/lib/analyze.js:241:5)
      at async run (/home/runner/work/_actions/github/codeql-action/v1/lib/analyze-action.js:74:9)
      at async runWrapper (/home/runner/work/_actions/github/codeql-action/v1/lib/analyze-action.js:201:9)
```

--- a/.github/workflows/codeql-analysis.yml
+++ b/.github/workflows/codeql-analysis.yml
@@ -32,7 +32,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        language: [ 'javascript', 'python' ]
+        language: [ 'python' ]
         # CodeQL supports [ 'cpp', 'csharp', 'go', 'java', 'javascript', 'python', 'ruby' ]
         # Learn more about CodeQL language support at https://git.io/codeql-language-support
 

